packages/markers/src/browser/problem/problem-tree-model.spec.ts

```diff
@@ -158,19 +167,20 @@ function createMockMarkerNode(marker: Marker<Diagnostic>): MarkerNode {
  * Create a mock diagnostic marker.
  * @param range the diagnostic range.
  * @param severity the diagnostic severity.
+ * @param owner the optional owner of the diagnostic
  *
  * @returns a mock diagnostic marker.
  */
-function createMockMarker(range: Range, severity: DiagnosticSeverity): Readonly<Marker<Diagnostic>> {
+ function createMockMarker(range: Range, severity: DiagnosticSeverity, owner?: string): Readonly<Marker<Diagnostic>> {
     const data: Diagnostic = {
         range: range,
         severity: severity,
-        message: 'message'
+        message: 'message',
     };
     return Object.freeze({
-        uri: name,
+        uri: '',
```

Comment:
This would also be fine but it's up to you:

```suggestion
        uri: 'uri',
```